giraffe/src/style/singleStatStyles.ts

```diff
@@ -1,10 +1,9 @@
-import CSS from 'csstype'
-
+import {CSSProperties} from 'react'
```

Comment:
For our purposes, there is virtually no difference between `CSS.Properties` from the  'csstype' package and `CSSProperties` from 'react'. Since there is no difference, for consistency, it is better to use `CSSProperties` everywhere rather than having isolated usages of `CSS.Properties` for no apparent reason.

Reply:
do we still need `csstype` as a dependency after these changes?

Reply:
We do not. Strangely, it is not even in the `package.json` to begin with. It's present in `yarn.lock` as a dependency of a dependency.